libcontainer/apparmor/apparmor.go

```diff
@@ -13,10 +14,8 @@ import (
 // IsEnabled returns true if apparmor is enabled for the host.
 func IsEnabled() bool {
 	if _, err := os.Stat("/sys/kernel/security/apparmor"); err == nil && os.Getenv("container") == "" {
```

Comment:
As suggested in the comments above, maybe we need to drop the `container` env var check (as a separate commit)